geo/src/main/java/org/odk/collect/geo/selection/SelectionSummarySheet.kt

```diff
@@ -47,6 +49,26 @@ internal class SelectionSummarySheet(context: Context, attrs: AttributeSet?) :
     fun setItem(item: MappableSelectItem) {
         itemId = item.id
 
+        when (item.status) {
+            Status.ERRORS -> {
+                binding.statusChip.visibility = View.VISIBLE
+                binding.statusChip.setIcon(org.odk.collect.icons.R.drawable.ic_baseline_rule_24)
```

Comment:
Maybe we could pull out a new `ErrorsPill` view for this and the instance list that could hold the logic for displaying the the "errors" or "no errors" state. Then you could just have something like:

```kotlin
when (item.status) {
    Status.ERRORS -> binding.errorsPill.errors = true
    Status.NO_ERRORS -> binding.errorsPill.errors = false
    else -> binding.errorsPill.visibility = View.GONE
}
```

Reply:
YEah, I was thinking about it but somehow didn't do that. Ok it's done.